benchmark/__main__.py

```diff
@@ -0,0 +1,20 @@
+"""Run the benchmarks for the given scenario."""
+
+import logging
+
+from benchmark.domain import BENCHMARKS
+from benchmark.result.result import Result
+
+
+def main():
+    """Run the performance test for the given scenario."""
```

Comment:
Not "given", as nothing is given here. Rather "defined", as this just takes the existing benchmarks. Also, plural (scenarioS)